lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -825,21 +825,20 @@ function getTimeSeriesChartContext(catalogItem, feature, getChartDetails) {
     CustomComponent.isRegistered("chart")
   ) {
     const chartDetails = getChartDetails();
-    const { title, csvData } = chartDetails;
     const distinguishingId = catalogItem.dataViewId;
     const featureId = defined(distinguishingId)
       ? distinguishingId + "--" + feature.id
       : feature.id;
     if (chartDetails) {
+      const { title, csvData } = chartDetails;
       const result = {
         ...chartDetails,
         id: featureId?.replace(/\"/g, ""),
         data: csvData?.replace(/\\n/g, "\\n")
       };
       const idAttr = 'id="' + result.id + '" ';
-      const sourceAttr = 'sources="1"';
       const titleAttr = title ? `title="${title}"` : "";
-      result.chart = `<chart ${idAttr} ${sourceAttr} ${titleAttr}>${result.data}</chart>`;
```

Comment:
There is no point passing `source="1"` as source. Instead now, when no source or download is specified, `ChartCustomComponent` will construct a downloadable link from the chart data.